src/charm.py

```diff
@@ -245,6 +244,19 @@ def init_server(self):
         if self.unit.is_leader() and not self.state.opensearch_server:
             self.app.status = BlockedStatus(MSG_DB_MISSING)
 
+    def _scrape_config(self) -> list[dict]:
+        """Generates the scrape config as needed."""
+        return [
+            {
+                "metrics_path": "/metrics",
+                "static_configs": [
+                    {"targets": [f"{self.state.unit_server.private_ip}:{COS_PORT}"]}
+                ],
+                # "tls_config": {"ca": self.state.unit_server.ca},
+                "scheme": "http",
```

Comment:
Should we add a conditional statement here? when TLS is enabled vs when it's not?

Reply:
`kibana-exporter` is only available HTTP :-/

Reply:
(NOTE: The whole COS service alltogether is only over HTTP, meaning Prometheus, Kibana, etc. interfaces. So I think w'er not that bad with this :-) Even thought indeed it's a limitation.)